### PR TITLE
Improve API and FHIR docs

### PR DESCRIPTION
**Pull request by prondubuisi**
Remove Licence from API and FHIR docs
Fix broken services and controller links for both docs
Add sections to API docs for easy navigation
Remove FHIR docs from API docs
Add single refrence for FHIR docs in API docs

Fixes issues #3361 #3366

### CODE REVIEW

**Comment by stu01509** [comment]
Now, We can say goodbye to the 404 pages 😁

**Comment by prondubuisi** [comment]
> Now, We can say goodbye to the 404 pages

Yes man!

File: API_README.md [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+### Sections
+* [facility API](https://github.com/openemr/openemr/blob/master/API_README.md#post-apifacility)
+* [provider API](https://github.com/openemr/openemr/blob/master/API_README.md#get-apiprovider)
+* [patient API](https://github.com/openemr/openemr/blob/master/API_README.md#post-apipatient)
+* [insurance API](https://github.com/openemr/openemr/blob/master/API_README.md#get-apipatientpidinsurance)
+* [appointment API](https://github.com/openemr/openemr/blob/master/API_README.md#get-apiappointment)
+* [document API](https://github.com/openemr/openemr/blob/master/API_README.md#get-apipatientpiddocument)
+* [message API](https://github.com/openemr/openemr/blob/master/API_README.md#post-apipatientpidmessage)
+* [dev notes](https://github.com/openemr/openemr/blob/master/API_README.md#dev-notes)
+* [todos](https://github.com/openemr/openemr/blob/master/API_README.md#project-management)
```
Comment (bradymiller) [inline API_README.md:16]: Having anchors is a good idea, but need to do them without requiring full path like this (or things will break when have this in different release branches). Is there a way to do this without requiring the full paths?

Comment (prondubuisi) [reply API_README.md:16]: Yes there is @bradymiller . I just need to add the relative file paths. Thanks.

Comment (prondubuisi) [reply API_README.md]: Hello @bradymiller, fixed

**Comment by bradymiller** [comment]
hi @prondubuisi , Looks great and bringing into codebase. Thanks for the nice docs! -brady

**Comment by sjpadgett** [comment]
I second. Thanks for this @prondubuisi .

**Pull request merged**